### ISSUE TITLE
DMP device doesn't work properly at least in the 780 (can't establish ddcmp DECNET circuit)

### ISSUE THREAD

**Issue by jguillaumes** [simh/simh#951]
The DMC simulation doesn't work in the current master. It is not possible to establish a DDCMP DECNET circuit between two simulators using their respective DMC interfaces. It is also not possible to establish a ddcmp circuit between a DMC and a DUP device, although the current DUP works with old DMCs.

- Last good commit:  337062fcf0aecbb6acf2ef5242e9faf330347af2 (Jun 13, 2020 05:11 CET)
- First bad commit: c8389a7bfd5aeb3ede9fb946ff7661537303660c (Jun 13, 2020 08:58 CET)

The comment in the "bad" commit is:
```
TMXR: Properly control and record modem bits on serial lines
    As reported in #877 and #878
```
So it is related to the serial lines, which seems to be consistent with a DMC problem.

First "bad" version show version output:

```
btes62>>  show ver
VAX 11/780 simulator V4.0-0 Current
    Simulator Framework Capabilities:
        64b data
        64b addresses
        Threaded Ethernet Packet transports:PCAP:TAP:VDE:NAT:UDP
        Idle/Throttling support is available
        Virtual Hard Disk (VHD) support
        RAW disk and CD/DVD ROM support
        Asynchronous I/O support (Lock free asynchronous event queue)
        Asynchronous Clock support
        FrontPanel API Version 12
    Host Platform:
        Compiler: GCC 6.3.0 20170516
        Simulator Compiled as C arch: x64 (Release Build) on Nov  1 2020 at 22:35:48
        Build Tool: simh-makefile
        Memory Access: Little Endian
        Memory Pointer Size: 64 bits
        Large File (>2GB) support
        SDL Video support: No Video Support
        No RegEx support for EXPECT commands
        OS clock resolution: 1ms
        Time taken by msleep(1): 1ms
        Ethernet packet info: libpcap version 1.8.1
        OS: Linux emult0 4.15.18-30-pve #1 SMP PVE 4.15.18-58 (Fri, 12 Jun 2020 13:53:01 +0200) x86_64 GNU/Linux
        git commit id: c8389a7b
        git commit time: 2020-06-13T08:58:38-0700
btes62>>
```
I can provide two VMS 4.7 images ready to reproduce the problem if they can help to find the fix.

**Comment by markpizz** [comment]
Please put the 2 disk images and 2 related configuration files zipped someplace (a Google Drive) that I can easily pick them up.

Thanks for the git bisect and the demonstration case.

**Comment by jguillaumes** [comment]
I've sent the link to the images via email and google invitation.

I've made an additional test. Since I found the problem linking a vax780 to a pdp10, and did the bisecting trying to connect an "old" (with a working DMC) vax780 to the "new" ones, there was the possibility of the "new" DMC working just against other "new" DMCs and not "old" DMCs or DUPs. The result has been negative: two vax780 running the HEAD version cannot talk betweem them.
